spec/system/admin/configuration/shipping_categories_spec.rb

```diff
@@ -54,8 +54,10 @@
       within row do
         expect(page).to have_content "Yes"
       end
+      category.reload
 
       expect(page).not_to have_content("Regular")
+      expect(category.name).to eq("Express")
```

Comment:
FYI you can reload the category directly  ie : 
```suggestion
      expect(category.reload.name).to eq("Express")
```
Anyway it's good like this as well.

Reply:
Thanks @rioug , will keep in mind further.